src/services/types.ts

```diff
@@ -474,7 +474,7 @@ namespace ts {
 
         /*@internal*/
         // eslint-disable-next-line @typescript-eslint/unified-signatures
-        getDefinitionAtPosition(fileName: string, position: number, searchOtherFilesOnly: boolean): readonly DefinitionInfo[] | undefined;
+        getDefinitionAtPosition(fileName: string, position: number, searchOtherFilesOnly: boolean, skipAlias: boolean): readonly DefinitionInfo[] | undefined;

```

Comment:
I'm not sure this is the best way to add this new parameter, open to feedback on this

Reply:
Small nit but I think I would make it `stopAtAlias` so you can just make it optional instead of defaulting to `true`. But wait until the end of the PR review to do this, because the implication of my other feedback may be that not all aliases should be stopped at / skipped.